scripts/obtain/dotnet-install.sh

```diff
@@ -115,6 +115,24 @@ get_current_os_name() {
     if [ "$uname" = "Darwin" ]; then
         echo "osx"
         return 0
+    else
+        if [ "$uname" = "Linux" ]; then
+            echo "linux"
+            return 0
+        fi
+    fi
+    
+    say_err "OS name could not be detected: $ID.$VERSION_ID"
+    return 1
+}
+
+get_distro_specific_os_name() {
```

Comment:
We aren't producing distro-specific archives for the runtime anymore.  And we should probably stop doing that for the SDK as well.  I think all this "distro-specific"/"alt-link" functionality can be removed.